tmt/steps/prepare/install.py

```diff
@@ -239,10 +247,36 @@ def go(self, guest):
 
             # Use both dnf install/reinstall to get all packages refreshed
             # FIXME Simplify this once BZ#1831022 is fixed/implemeted.
-            guest.execute(f"{command} install {options} {rpms_directory}/*")
             if 'rpm-ostree' not in command:
+                guest.execute(f"{command} install {options} {rpms_directory}/*")
                 guest.execute(f"{command} reinstall {options} {rpms_directory}/*")
-            summary = fmf.utils.listed(local_packages, 'local package')
+                local_packages_installed.append(local_packages)
+            else:
+                for package in local_packages:
+                    check = f'dnf list installed {os.path.basename(package)}'
+                    if 'x86_64' in check:
+                        check = check.strip('x86_64.rpm')
+                    elif 'aarch64' in check:
+                        check = check.strip('aarch64.rpm')
+                    else:
+                        self.debug(f"Local rpm package arch not supported", level=3)
+                        continue
+                    try:
+                        guest.execute(f"{check}")
+                        self.debug(f"Local rpm:"
+                                   f"Package: {package} already installed.", level=3)
+                    except tmt.utils.RunError:
+                        try:
+                            guest.execute(f"{command} install {options} "
+                                          f"{rpms_directory}/{os.path.basename(package)}")
+                            local_packages_installed.append(package)
+                        except tmt.utils.RunError as e:
+                            self.debug("Local rpm:"
+                                       f"Package: {package} not installed."
+                                       f"err: {e.stderr}", level=3)
```

Comment:
Would it probably make sense to use `self.warn()` instead to inform user about possible installation problems?